team-meeting-notes/2020-12-10.md

```diff
@@ -0,0 +1,60 @@
+# Package team meeting [DATE]
+
+Attendees: Dave, David, Jesper, Kevin, Lotte & Richard
+
+Notetaker: Jesper
+
+## Agenda
+
+- Status on Umbpack
+- Status on Package Templates
+- Status on Umbraco.Packages issues
+- Status on UI Examples
+- Any other business
+- Tasks for next time
+
+## Status on Umbpack
+
+No changes
+
+## Status on Package Templates
+
+We've merged in a [property editor](https://github.com/umbraco/Package.Templates/pull/26) itemtemplate and also a [custom tree](https://github.com/umbraco/Package.Templates/pull/34) example. There are a few further improvements we can add to the tree in the future.
+With these 2 new item templates and the [Visual Studio template meta data improvements](https://github.com/umbraco/Package.Templates/pull/29) we are ready to create another release soon 🎉
+
+## Status on Umbraco.Packages issues
+
+Merged a few smaller Our prs, we will have an upcoming problem when uSync hits 1 million downloads, some styling will need to be updated to accomodate such a massively successful package! 
```

Comment:
If you want to reference the issue for the "1 million problem" it's https://github.com/umbraco/Umbraco.Packages/issues/107